modules/maintainers/auxiliaries/deputize/request.pb.go

```diff
@@ -0,0 +1,576 @@
+	ToID                 github_com_persistenceOne_persistenceSDK_schema_types.ID         `protobuf:"bytes,2,opt,name=toID,proto3,customtype=github.com/persistenceOne/persistenceSDK/schema/types.ID" json:"toID"valid:"required~required field toID missing"`
+	ClassificationID     github_com_persistenceOne_persistenceSDK_schema_types.ID         `protobuf:"bytes,3,opt,name=classificationID,proto3,customtype=github.com/persistenceOne/persistenceSDK/schema/types.ID" json:"classificationID"valid:"required~required field classificationID missing"`
+	MaintainedProperties github_com_persistenceOne_persistenceSDK_schema_types.Properties `protobuf:"bytes,4,opt,name=maintainedProperties,proto3,customtype=github.com/persistenceOne/persistenceSDK/schema/types.Properties" json:"maintainedProperties"valid:"required~required field maintainedProperties missing"`
+	AddMaintainer        bool                                                             `protobuf:"varint,5,opt,name=addMaintainer,proto3" json:"addMaintainer,omitempty"valid:"required~required field addMaintainer missing"`
+	RemoveMaintainer     bool                                                             `protobuf:"varint,6,opt,name=removeMaintainer,proto3" json:"removeMaintainer,omitempty"valid:"required~required field removeMaintainer missing"`
+	MutateMaintainer     bool                                                             `protobuf:"varint,7,opt,name=mutateMaintainer,proto3" json:"mutateMaintainer,omitempty"valid:"required~required field mutateMaintainer missing"`
+}
+
+func (m *auxiliaryRequest) Reset()         { *m = auxiliaryRequest{} }
+func (m *auxiliaryRequest) String() string { return proto.CompactTextString(m) }
+func (*auxiliaryRequest) ProtoMessage()    {}
+func (*auxiliaryRequest) Descriptor() ([]byte, []int) {
+	return fileDescriptor_1e088071cb39a9ad, []int{0}
+}
+func (m *auxiliaryRequest) XXX_Unmarshal(b []byte) error {
+	return m.Unmarshal(b)
+}
+func (m *auxiliaryRequest) XXX_Marshal(b []byte, deterministic bool) ([]byte, error) {
+	if deterministic {
+		return xxx_messageInfo_AuxiliaryRequest.Marshal(b, m, deterministic)
+	} else {
+		b = b[:cap(b)]
+		n, err := m.MarshalToSizedBuffer(b)
+		if err != nil {
+			return nil, err
+		}
+		return b[:n], nil
+	}
+}
+func (m *auxiliaryRequest) XXX_Merge(src proto.Message) {
+	xxx_messageInfo_AuxiliaryRequest.Merge(m, src)
+}
+func (m *auxiliaryRequest) XXX_Size() int {
+	return m.Size()
+}
+func (m *auxiliaryRequest) XXX_DiscardUnknown() {
+	xxx_messageInfo_AuxiliaryRequest.DiscardUnknown(m)
+}
+
+var xxx_messageInfo_AuxiliaryRequest proto.InternalMessageInfo
+
+func init() {
+	proto.RegisterType((*auxiliaryRequest)(nil), "persistence.modules.maintainers.auxiliaries.deputize.v1beta1.auxiliaryRequest")
+}
+
+func init() {
+	proto.RegisterFile("proto/persistence/modules/maintainers/auxiliaries/deputize/v1beta1/request.proto", fileDescriptor_1e088071cb39a9ad)
+}
+
+var fileDescriptor_1e088071cb39a9ad = []byte{
```

Comment:
naming! just rename to fileDescriptor, move to constants or find a way to programmatically generate